.github/renovate.json

```diff
@@ -15,5 +15,6 @@
   "rebaseWhen": "never",
   "schedule": ["every weekend"],
   "prConcurrentLimit": 1,
-  "prHourlyLimit": 1
+  "prHourlyLimit": 1,
+  "ignorePaths": ["src/mobile/**"]
```

Comment:
Do we need ** after / ?

Reply:
I think so, I followed the [docs](https://docs.renovatebot.com/configuration-options/#ignorepaths). This is hard to know if this will work until we have it in `master`

Reply:
@cindyledev [I think so too, let's check](https://github.com/renovatebot/config-help/issues/229)

Reply:
I think both would work